### ISSUE TITLE
Please add translation support

### ISSUE THREAD

**Issue by yar0d** [script-wizards/oracle#7]
Hello,

It would be great if you could i18n*l10n all your applications. I will contribute to translate to French.

**Comment by ptaranat** [comment]
I agree, this is a good feature. I would like to save this for v1.0.0 release as the interface may drastically change. Fortunately for Oracle the UI elements that deserve i18n should be few.
